AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/ValidatorHelper.java

```diff
@@ -168,8 +170,8 @@ public void enclosingElementHasEActivityOrEFragment(Element element, ElementVali
 		enclosingElementHasOneOfAnnotations(element, validAnnotations, valid);
 	}
 
-	public void enclosingElementHasEActivityOrEFragmentOrEServiceOrEIntentService(Element element, ElementValidation valid) {
-		List<Class<? extends Annotation>> validAnnotations = asList(EActivity.class, EFragment.class, EService.class, EIntentService.class);
+	public void enclosingElementHasEActivityOrEFragmentOrEServiceOrEIntentServiceOrEViewOrEViewGroup(Element element, ElementValidation valid) {
```

Comment:
okay... just realized that all places where this is called are necessarily part of this PR and that it is only used in one place. ignore this comment :D